modules/local/input_assure/main.nf

```diff
@@ -11,19 +10,19 @@ process INPUT_CHECK{
     tuple val(meta), path(mlst)
 
     output:
-    tuple val(meta), path("${meta.id}_match.txt"), path(mlst),      emit: match
+    tuple val(meta), path("${meta.id}.mlst.json"),                  emit: result
     tuple val(meta), path("*_error_report.csv"), optional: true,    emit: error_report
     path("versions.yml"),                                           emit: versions
 
     script:
 
     """
-    input_check.py \\
+    input_assure.py \\
         --input ${mlst} \\
         --sample_id ${meta.id} \\
         --address ${meta.address} \\
         --output_error ${meta.id}_error_report.csv \\
-        --output_match ${meta.id}_match.txt
+        --output_json ${meta.id}.mlst.json
```

Comment:
Could you change to output gzipped JSON.

Reply:
Done here 😸  : [348fe95](https://github.com/phac-nml/gasnomenclature/pull/13/commits/348fe9558c27e1ef76ba387c4abc31632656c1aa)